Night CI 2024-11-15: Failed

action url: https://github.com/spidernet-io/spiderpool/actions/runs/11862720387

```
E: Failed to fetch http://deb.debian.org/debian/pool/main/p/python3.9/libpython3.9-minimal_3.9.2-1_amd64.deb  Connection timed out [IP: 199.232.98.132 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
make[2]: *** [Makefile:35: kind-init] Error 100
make[2]: Leaving directory '/home/runner/work/spiderpool/spiderpool/test'
make[1]: *** [Makefile:299: e2e_init] Error 2
make[1]: Leaving directory '/home/runner/work/spiderpool/spiderpool'
make: *** [Makefile:331: e2e_init_calico] Error 2
```

--- a/test/scripts/install-ovs.sh
+++ b/test/scripts/install-ovs.sh
@@ -66,21 +66,24 @@ for NODE in $KIND_NODES; do
   install_openvswitch() {
     for attempt in {1..5}; do
       echo "Attempt $attempt to install openvswitch on ${NODE}..."
-      docker exec ${NODE} apt-get update > /dev/null
-      docker exec ${NODE} apt-get install -y apt-utils > /dev/null
-      docker exec ${NODE} apt-get install -y openvswitch-switch > /dev/null
-      
-      if [[ $? -eq 0 ]]; then
-        echo "Openvswitch installed successfully on ${NODE}"
-        return 0
+      if ! docker exec ${NODE} apt-get update > /dev/null; then
+        echo "Failed to update package list on ${NODE}, retrying in 10s..."
+        sleep 10
+        continue
       fi
-
-      echo "Failed to install openvswitch on ${NODE}, retrying in 10s..."
-      sleep 10
+      
+      if ! docker exec ${NODE} apt-get install -y openvswitch-switch > /dev/null; then
+        echo "Failed to install openvswitch on ${NODE}, retrying in 10s..."
+        sleep 10
+        continue
+      fi
+      
+      echo "Succeed to install openvswitch on ${NODE}"
+      return 0
     done
-    
-    echo "Error: Failed to install openvswitch on ${NODE} after 5 attempts"
-    exit 1
+
+    echo "Error: Failed to install openvswitch on ${NODE} after 5 attempts." >&2
+    return 1
   }
   
   echo "=========install openvswitch"

